src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/LocalDBTest/LocalDBTest.cs

```diff
@@ -13,14 +15,18 @@ public static class LocalDBTest
         private static readonly string s_localDbConnectionString = @$"server=(localdb)\{DataTestUtility.LocalDbAppName}";
         private static readonly string[] s_sharedLocalDbInstances = new string[] { @$"server=(localdb)\.\{DataTestUtility.LocalDbSharedInstanceName}", @$"server=(localdb)\." };
         private static readonly string s_badConnectionString = $@"server=(localdb)\{DataTestUtility.LocalDbAppName};Database=DOES_NOT_EXIST;Pooling=false;";
+        private static readonly string s_commandPrompt = "cmd.exe";
+        private static readonly string s_sqlLocalDbInfo = @$"/c SqlLocalDb info {DataTestUtility.LocalDbAppName}";
+        private static readonly string s_startLocalDbCommand = @$"/c SqlLocalDb start {DataTestUtility.LocalDbAppName}";
+        private static readonly string s_localDbNamedPipeConnectionString = @$"server={GetLocalDbNamedPipe()}";
```

Comment:
Should we test for local db names with leading white space?

Reply:
@johnnypham Are you referring to space between /c SqlLocalDb ? This is probably because of https://stackoverflow.com/questions/515309/what-does-cmd-c-mean

Reply:
so, with adding TrimStart I dont think we need to test for leading whitespaces. Correct?

Reply:
not for local db, but we have tests for leading spaces in general as server name.